### ISSUE TITLE
select

### ISSUE THREAD

**Issue by pax3701** [AIGODLIKE/gesture_helper#22]
auto select while i still holding my mouse button

**Comment by xmx-emm** [comment]
Unable to understand the problem from the description

You can check this option
![image](https://github.com/user-attachments/assets/565ccc1f-d235-4839-9bab-c09320440f3d)

**Comment by pax3701** [comment]
yes im uncheck that, sometimes when draw gesture my mouse button not release but it still select
and i cant choose gesture preview button on my N panel
sorry my english not good

**Comment by xmx-emm** [comment]
Maybe we can record a video
The execution method is that if a gesture is displayed, it needs to be released and there is an activity item before it can be executed

**Comment by pax3701** [comment]
Thanks, this happens quite rarely and only causes a slight inconvenience while working, so I will try to record a video if possible.

**Comment by xmx-emm** [comment]
Then you can open debug, and if it happens again, you can send the console log for analysis
![image](https://github.com/user-attachments/assets/30a17f90-425f-4b94-967e-103a3ebf75cd)

**Comment by pax3701** [comment]
https://github.com/user-attachments/assets/0ec7224e-5cd2-4e01-b479-fe2308bf0bc6

video im record today

**Comment by xmx-emm** [comment]
There is no obvious bug in the video, perhaps you can adjust the timeout to 300ms to see if it still triggers

**Comment by xmx-emm** [comment]
2.0.9 has been released, please try if there is still this issue
﻿
https://github.com/AIGODLIKE/gesture_helper/releases/tag/v2.0.9

**Comment by pax3701** [comment]
i think issue from nvidia, my old gtx 1650 still good, but my 3060 still have issue
sorry my english not good

**Comment by xmx-emm** [comment]
Strange issues, perhaps you can try troubleshooting by using an old version plugin to verify if a certain version is starting to have problems

**Comment by pax3701** [comment]
Hello, I have an idea. Can you add a "press to choose" option in the addon?

**Comment by pax3701** [comment]
What I mean is a pie menu like the current one, but it only responds when clicking the mouse. Clicking to select a command gives me a more control, and I think it would also make it easier to integrate more than 8 commands.
I’m just giving feedback. Currently, I’m using 4 out of your 6 addons, and I love them
my english not good, sorry

**Comment by xmx-emm** [comment]
Perhaps this method is feasible, and this feature will be added in future updates

**Comment by pax3701** [comment]
Hello, today I just discovered a software for pie menus on Windows called AutoHotPie. Do you know about it? I find it quite interesting.

**Comment by xmx-emm** [comment]
A great tool, but it cannot be used in Blender
